app/containers/Products/FutureBenchmarks.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {RelayModernEnvironment} from 'relay-runtime/lib/store/RelayModernEnvironment';
+import JsonSchemaForm, {IChangeEvent} from 'react-jsonschema-form';
+import {Button, Container} from 'react-bootstrap';
+import moment from 'moment';
+import FormFieldTemplate from '../Forms/FormFieldTemplate';
+import FormObjectFieldTemplate from '../Forms/FormObjectFieldTemplate';
+
+import editBenchmarkMutation from '../../mutations/benchmark/editBenchmarkMutation';
+import {
+  benchmarkSchema,
+  benchmarkUISchema
+} from './ProductBenchmarkJsonSchemas';
+
+interface Props {
+  benchmark: any;
+  environment: RelayModernEnvironment;
+  deleteBenchmark: (...args: any[]) => void;
+}
+
+const FutureBenchmarksComponent: React.FunctionComponent<Props> = ({
+  benchmark,
+  environment,
```

Comment:
Not a big fan of passing the relay environment through props. Either the update should be done in the parent component via an event handler, or this should be a fragment container

Reply:
fixed with a handler function